### ISSUE TITLE
Do you have a plan to improve native UI's with adding to some of UI kits like Material Design or somethi'?

### ISSUE THREAD

**Issue by FFC12** [AshampooSystems/boden#11]
I'm just wondering to what's your plans about some features like UI and animations of itself. As Boden evolves, will you make improvements to the application look in the future using ui libraries such as **Material design**?

**Comment by gitoby** [comment]
Yes, we will support Material Design components for Android at some point in the future. It's not yet 100% clear how Boden will support different UI libraries, but it's definitely a goal to support the most popular modern design languages.
